demo/src/main/java/com/novoda/demo/MainActivity.java

```diff
@@ -85,24 +88,34 @@ public void onClick(View v) {
 
         private void showVideoSelectionDialog() {
             final List<PlayerVideoTrack> videoTracks = player.getVideoTracks();
+
             ArrayAdapter<String> adapter = new ArrayAdapter<>(MainActivity.this, R.layout.list_item);
             adapter.addAll(mapVideoTrackToLabel(videoTracks));
             AlertDialog videoTrackSelectionDialog = new AlertDialog.Builder(MainActivity.this)
                     .setTitle("Select Video track")
                     .setAdapter(adapter, new DialogInterface.OnClickListener() {
                         @Override
                         public void onClick(DialogInterface dialog, int position) {
-                            PlayerVideoTrack videoTrack = videoTracks.get(position);
-                            player.selectVideoTrack(videoTrack);
+                            switch (position) {
+                                case 0:
```

Comment:
should this be an if else?

Reply:
Yea it should 😆  this is what I get for reusing some of our code.